### DISCUSSION TITLE
Make a sincere request for help about conda

### DISCUSSION THREAD

**Discussion by ZJYZXH** [idaholab/moose#19216] [Q&A Getting Started] [answered]
x86_64-conda-linux-gnu-c++: fatal error: Killed signal terminated program cc1plus

**Comment by GiudGiud** [comment]
Hello

Can you please post more context

Guillaume

**Reply by ZJYZXH** [reply]
Please  wait  a  moment  .  It  is  morning  now  and  I  have  a  class  .  I  will  reply  you  later,   ok?

**Reply by ZJYZXH** [reply]
Please help  me!!

**Comment by loganharbour** [comment, marked as answer]
Are you running this in a virtual machine? And if so, what resources do you have allocated to it?

**Reply by ZJYZXH** [reply]
I installed  the  Ubuntu  system  in  the  virtual  machine  , and  then  ran  it  in  the  terminal  according  to  the  official  website  instructions.  Finally,  this  error  occurred  in  the  test.

**Reply by loganharbour** [reply]
What resources did you give the virtual machine (memory/cpus/storage)?

**Reply by ZJYZXH** [reply]
4Gb

**Reply by GiudGiud** [reply]
4GB ram or storage

**Reply by ZJYZXH** [reply]
4GB  ram
20GB   storage

**Comment by dschwen** [comment]
I found  a good rule of thumb to be to have at most one concurrent compile process per GB of available RAM. Meaning you should run `make -j 4` on your VM. The killed compiler is 99% certainly an out of memory issue.

**Reply by ZJYZXH** [reply]
Run  the  conmand  again?

**Reply by GiudGiud** [reply]
did you run with a -j argument the first time?

**Reply by ZJYZXH** [reply]
yes

**Reply by GiudGiud** [reply]
try running just make then. It ll use less resources

**Reply by ZJYZXH** [reply]
first   time   run   with    make  -j   4

**Reply by dschwen** [reply]
Do you have swap set up? Either way try `make -j 2`

**Comment by ZJYZXH** [comment]


**Reply by GiudGiud** [reply]
cant see the screenshot

**Comment by ZJYZXH** [comment]


**Reply by GiudGiud** [reply]
still cant see it
Can you please post it from github rather than email it?

**Reply by ZJYZXH** [reply]
![屏幕截图 2021-10-27 135150](https://user-images.githubusercontent.com/91814320/139354473-bf1e19f6-a522-4f92-be2c-90a2f2b33079.png)

**Reply by ZJYZXH** [reply]
![屏幕截图 2021-10-27 144213](https://user-images.githubusercontent.com/91814320/139354508-991031fc-59f4-4aab-93fa-cfc50007df7e.png)

**Reply by ZJYZXH** [reply]
Can you  see  it?   And   it   successful?

**Reply by GiudGiud** [reply]
looks successful to me!
It compiled and it passed the test for your application.

Does it pass the tests in moose/test?

**Reply by ZJYZXH** [reply]
Most  of  them  are  ok,  but  a  few  of  them  have  been  skipped,  no  mistakes.

**Reply by GiudGiud** [reply]
You're all set then. Good luck with your modeling!

**Reply by ZJYZXH** [reply]
thanks

**Reply by ZJYZXH** [reply]
Hello, I have a new question to ask you, the first example I run is normal, but as shown in the picture below, the fourth and other examples all report the same error, why is this?

**Reply by ZJYZXH** [reply]
![屏幕截图 2021-10-31 112444](https://user-images.githubusercontent.com/91814320/139566075-e38cfc3a-7a2e-4896-be8c-b502bb5a8961.png)

**Reply by GiudGiud** [reply]
That file is not in that directory?
i just ran ls -l there:
```
-rw-rw-r-- 1 guillaume guillaume 1469 Mar  4  2021 dirichlet_bc.i
-rw-rw-r-- 1 guillaume guillaume 1548 Oct 11 22:03 Makefile
-rw-rw-r-- 1 guillaume guillaume 1467 Mar  4  2021 neumann_bc.i
-rw-rw-r-- 1 guillaume guillaume 1165 Mar  4  2021 periodic_bc.i
-rw-rw-r-- 1 guillaume guillaume 2432 Mar  4  2021 square.e
-rw-rw-r-- 1 guillaume guillaume  230 Mar  4  2021 training_list.txt
-rw-rw-r-- 1 guillaume guillaume 8964 Mar  4  2021 trapezoid.e
```

**Reply by GiudGiud** [reply]
why do you want that file to exist? Is it mentioned in the tutorial?
I dont see it there
https://mooseframework.inl.gov/getting_started/examples_and_tutorials/examples/ex04_bcs.html

**Reply by ZJYZXH** [reply]
I tried to run the other cases in the moose folder, but couldn't do it. The previous one was ex04.i, but failed.

**Reply by GiudGiud** [reply]
what error do you get?
for neumann_bc.i for example. 

Also please dont post screenshots. they are not helpful when using the search feature. It's easier to copy paste the terminal output inside triple quotes (for formatting) anyway

**Reply by YaqiWang** [reply]
Looks like the installation issue has been resolved. You need to start a new conversation for new issues.

**Comment by ZJYZXH** [comment]


**Reply by GiudGiud** [reply]
hey @ZJYZXH and @amosaha. Please refrain from posting in Chinese here. While I can read this simple exchange of contact details, most of the community cannot so it's a little uncomfortable for the next person who will consult this post looking for help on the same topic.

**Reply by amosaha** [reply]
I‘m so sorry, and I have deleted the messages.I won’t send such messages.Thank you for reminding me of this rule.

**Reply by GiudGiud** [reply]
Hey no worries we literally just added it to the guidelines this morning! The situation had not come up before. Good luck with using MOOSE!

**Comment by ZJYZXH** [comment]
This is the first time to do this, and my English is poor, I feel very sorry.




---Original---
From: "Guillaume ***@***.***&gt;
Date: Mon, Nov 1, 2021 23:40 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [idaholab/moose] Make a sincere request for help about conda(Discussion #19216)




 
Hey no worries we literally just added it to the guidelines this morning! The situation had not come up before. Good luck with using MOOSE!
 
—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
